data/beaconrestapi/src/main/java/tech/pegasys/teku/beaconrestapi/handlers/tekuv1/validatorInclusion/GetGlobalValidatorInclusion.java

```diff
@@ -0,0 +1,261 @@
+            .description("Returns a global count of votes for a given epoch.")
+            .tags(TAG_TEKU, TAG_EXPERIMENTAL)
+            .pathParam(EPOCH_PARAMETER)
+            .response(
+                SC_OK,
+                "Request successful",
+                SerializableTypeDefinition.<GetGlobalValidatorResponseData>object()
+                    .name("GetGlobalValidatorInclusionResponse")
+                    .withField(
+                        "data", GetGlobalValidatorResponseData.RESPONSE_DATA, Function.identity())
+                    .build())
+            .withNotFoundResponse()
+            .withServiceUnavailableResponse()
+            .build());
+    this.chainDataProvider = chainDataProvider;
+    this.spec = spec;
+    this.timeProvider = timeProvider;
+  }
+
+  @OpenApi(
+      path = OAPI_ROUTE,
+      method = HttpMethod.GET,
+      summary = "Get Global Validator Inclusion",
+      tags = {TAG_TEKU, TAG_EXPERIMENTAL},
+      description = "Returns a global count of votes for a given epoch.",
+      pathParams = {@OpenApiParam(name = "epoch", description = "Epoch to get data for")},
+      responses = {
+        @OpenApiResponse(
+            status = RES_OK,
+            content = {@OpenApiContent(type = JSON, from = GetGlobalValidatorResponse.class)}),
+        @OpenApiResponse(status = RES_BAD_REQUEST),
+        @OpenApiResponse(status = RES_NOT_FOUND),
+        @OpenApiResponse(status = RES_INTERNAL_ERROR),
+        @OpenApiResponse(
+            status = RES_SERVICE_UNAVAILABLE,
+            description = "Beacon node is currently syncing.")
+      })
+  @Override
+  public void handle(@NotNull final Context ctx) throws Exception {
+    adapt(ctx);
+  }
+
+  @Override
+  public void handleRequest(final RestApiRequest request) throws JsonProcessingException {
+    final UInt64 epoch = request.getPathParameter(EPOCH_PARAMETER);
+    request.respondAsync(
+        chainDataProvider
+            .getValidatorInclusionStateAtEpoch(epoch)
+            .thenApply(
+                maybeStateAndMetadata ->
+                    maybeStateAndMetadata
+                        .map(
+                            stateAndMetadata ->
+                                AsyncApiResponse.respondOk(
+                                    getResponseFromState(stateAndMetadata.getData())))
+                        .orElse(AsyncApiResponse.respondServiceUnavailable())));
+  }
+
+  private GetGlobalValidatorResponseData getResponseFromState(final BeaconState state) {
+    final UInt64 timeStart = timeProvider.getTimeInMillis();
+    final ValidatorStatuses validatorStatuses =
+        spec.atSlot(state.getSlot()).getValidatorStatusFactory().createValidatorStatuses(state);
+    final UInt64 timeEnd = timeProvider.getTimeInMillis();
+    LOG.debug(
+        "Time taken to compute validator statuses: " + timeEnd.minusMinZero(timeStart) + " ms");
+    return new GetGlobalValidatorResponseData(validatorStatuses.getTotalBalances());
+  }
```

Comment:
I feel like this is business logic that would be better in the chain data provider - particularly since we had to add a new method anyway.